build.sbt

```diff
@@ -1144,7 +1144,8 @@ lazy val `engine-runner` = project
       "commons-cli"         % "commons-cli"  % commonsCliVersion,
       "com.monovore"       %% "decline"      % declineVersion,
       "org.jline"           % "jline"        % jlineVersion,
-      "org.typelevel"      %% "cats-core"    % catsVersion
+      "org.typelevel"      %% "cats-core"    % catsVersion,
+      "com.google.guava"    % "guava"        % guavaVersion
```

Comment:
As indicated in the `Legal Review / Verify Notice Package` you need to run `enso/gatherLicenses` when updating dependencies to update the report state. You should not need to modify the review report as Guava was already a dependency, but the report state needs to be updated as the dependency list changed.

Reply:
done